ts/environment.ts

```diff
@@ -152,17 +163,18 @@ export function mkEnvironment(
 ): Environment {
   const copySource = (src: string) => nixStrLit`
     echo copying source
-    cp -r ${nixSource(src)} src
-    chmod -R u+rwX src
-    cd src
+    cp -r ${nixSource(src)}/. .
```

Comment:
The docstring above is outdated (missing `src` param, and has `mkEnvironment` not take a record)